packages/core/src/browser/connection-status-service.ts

```diff
@@ -198,9 +198,8 @@ export class ApplicationConnectionStatusContribution extends DefaultFrontendAppl
     private statusbarId = 'connection-status';
 
     protected handleOnline(): void {
-        this.statusBar.setBackgroundColor(undefined);
-        this.statusBar.setColor(undefined);
         this.statusBar.removeElement(this.statusbarId);
+        this.statusBar.removeCls('theia-state-offline');
```

Comment:
abbreviations are not allowed by our coding guideline: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#whole-words-names